frontend/src/translations/en.json

```diff
@@ -207,7 +207,7 @@
   "Welcome": {
     "title": "Welcome to Falco !",
     "no_project": "You don't have access to any project for now.",
-    "create_project": "To create a new project, please contact Jaafar (jaafarr@theodo.fr) or Nicolas (nicolasgo@theodo.fr).",
+    "create_project": "To create a new project, please contact Jaafar (jaafarr@theodo.fr), Nicolas (nicolasgo@theodo.fr) or François (francoisf@theodo.fr).",
```

Comment:
I think you can change this label altogether and redirect to the appropriate page in the docs!